package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "bookstore",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
```

Comment:
- [ ] I was unable to run your application from my local machine, see the screenshot below 👇🏻 looks like your package.json file is missing the start. Please correct this 💯 👍🏻

![image](https://user-images.githubusercontent.com/79058364/225638986-92156eac-6bd4-4847-8ffe-995164b99226.png)